tools/operator/connectionprofile/updateconncectionprofile.go

```diff
@@ -63,15 +63,43 @@ func (c ConnProfile) updateConnectionProfilePerChannel(inputObject interface{},
 func (c ConnProfile) updateConnectionProfilePerOrg(organizations []inputStructs.Organization, inputArgs ...string) error {
 
 	var err error
+	connectionProfilesList := []string{}
 	action, orgName, channelName := inputArgs[0], inputArgs[1], inputArgs[2]
 	connProfilePath := paths.GetConnProfilePathForOrg(orgName, organizations)
-	switch action {
-	case "create":
-		err = c.updateConnectionProfile(connProfilePath, channelName, "orderer")
-	case "join":
-		err = c.updateConnectionProfile(connProfilePath, channelName, "peer")
-	case "instantiate", "upgrade":
-		err = c.updateConnectionProfile(connProfilePath, channelName, "chaincodes", inputArgs[len(inputArgs)-1])
+	connectionProfilesList = append(connectionProfilesList, connProfilePath)
+	if !strings.HasSuffix(connProfilePath, ".yaml") && !strings.HasSuffix(connProfilePath, ".yml"){
+		connectionProfilesList = []string{}
+		currentDir, err := paths.GetCurrentDir()
+		if err != nil {
+			logger.ERROR("ConnectionProfile: Failed to get the current directory", connProfilePath)
```

Comment:
if you are going to share connProfilePath, which has little bearing on the error, then at least say what it is you are printing. Either delete the connProfilePath from this error log, or insert text, e.g.:  ...
    directory, connProfilePath:", connProfilePath)

Reply:
Done